### DISCUSSION TITLE
Welcome to wenda Discussions!

### DISCUSSION THREAD

**Discussion by l15y** [wenda-LLM/wenda#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by b7811433** [comment]
我认为增加一键切换模型，类似stable-diffusion-webui那样体验会更好

**Comment by Pumayou** [comment]
按介绍下载文件到本地。运行后出现两个错误：
#1 --- 错误Error communicating with OpenAI: HTTPConnectionPool(host=‘127.0.0.1’, port=8000): Max retries exceeded with url: /v1/chat/completions (Caused by NewConnectionError(‘<urllib3.connection.HTTPConnection object at 0x000001E629FB8090>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。’))

#2 --- 找不到库： plugins.common
